back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -63,6 +62,7 @@ def auth_callback(verify:, authver_method:)
     provider = auth['provider']
     user_attrs = authver_method.profile_to_user_attrs(auth)
 
+    @identity = Identity.find_or_build_with_omniauth(auth, authver_method)
```

Comment:
Just moved it because it makes more sense here.

Reply:
OK, assume that `@identity` can be set in previous calls to this method given that we are potentially returning it before it is set